lab-justin/lib/client-pool.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+const EventEmitter = require('events').EventEmitter;
+var uuid = require('node-uuid');
```

Comment:
If you use modules that aren't built into node, you must include a package.json with your assignment.